src/components/AppMain/DisplayCard/DisplayCard.tsx

```diff
@@ -29,7 +30,8 @@ const DisplayCard = ({ value }: DisplayCardProps) => {
 	const handleCancel = () => {
 		setIsEdit(Edit.False)
 	}
-	const handleEdit = () => {
+	const handleEdit = event => {
+		event.stopPropagation()
 		setIsEdit(Edit.True)
```

Comment:
**nit:** This makes no sense to set such enum `setIsEdit(Edit.True)`. What I previously meant is to hold card side in enum like `setCardSide(Side.Front)`. Currently your implementation does not differ from simple boolean use